features/rules/GRF/GRF000_Georeferencing.feature

```diff
@@ -0,0 +1,19 @@
+@implementer-agreement
+@GRF
+@version1
+@E00050
+Feature: GRF000 - Georeferencing
+The rule verifies the presence of IFC entities used define the geographic location and orientation of the model relative to a coordinate reference system.
+
+
+Background: 
+
+    Given A model with Schema 'IFC4.3' or 'IFC4'
+
+
+    Scenario: Georeferencing defined with a reference coordinate system
+
+        Given An .IfcCoordinateReferenceSystem.
```

Comment:
I'm a little bit in doubt here whether we should go for the complete Project > Context > CoordOp

But don't really have arguments in favour or against. Combined with the rule that prevents orphaned non-rooted instances, it's not really possible to connect a CRS in another way to the model.

Reply:
> But don't really have arguments in favour or against

Well, according to The Zen of Python, which @civilx64  often refers to: "Explicit is better than implicit."
; https://peps.python.org/pep-0020/#the-zen-of-python
So that would be: (?)
```
Given An IfcProject 
Given Its attribute RepresentationContexts
Given Its attribute HasCoordinateOperation
Given Its attribute TargetCRS
Then activate rule
```